CVARS.rst

```diff
@@ -450,6 +459,42 @@ Server-Side
 
 ..
 
+:Name: sv_maxRate
+:Valid: "0", Integer >= 1000
```

Comment:
Remove magic value 0 (see later)

Reply:
Not a new cvar, so it has to stay for consistency with existing setups (see later).